internal/pkg/deploy/cloudformation/stack/name.go

```diff
@@ -23,6 +36,6 @@ func NameForEnv(app, env string) string {
 }
 
 // NameForTask returns the stack name for a task.
-func NameForTask(task string) string {
-	return fmt.Sprintf("task-%s", task)
+func NameForTask(task string) TaskStackName {
+	return TaskStackName(fmt.Sprintf(fmtTaskStackName, task))
```

Comment:
Maybe we can ditch `fmtTaskStackName` since we also have `taskStackPrefix`.
```suggestion
	return TaskStackName(taskStackPrefix + task)
```